extensions-core/orc-extensions/src/test/java/org/apache/druid/data/input/orc/OrcReaderTest.java

```diff
@@ -314,8 +314,13 @@ public void testJsonPathFunctions() throws IOException
         //deviation of [7,8,9] is 1/3, stddev is sqrt(1/3), approximately 0.8165
         Assert.assertEquals(0.8165, Double.parseDouble(Iterables.getOnlyElement(row.getDimension("stddev"))), 0.0001);
 
-        //append is not supported
-        Assert.assertEquals(Collections.emptyList(), row.getDimension("append"));
+        //Support for append has been added in json-path-2.9.0 but it is unused
```

Comment:
maybe update this comment to say that append is not support with Orc format and link to the original PR where we mention that support level across formats. I think that will make it more explicit.

Reply:
```suggestion
        // we do not support json-path append function for ORC format (see https://github.com/apache/druid/pull/11722)
```

Reply:
cc @FrankChen021 any concerns here since you originally added support for this?